### ISSUE TITLE
[1.18.2]  Crash when producing certain sound

### ISSUE THREAD

**Issue by Reshy** [henkelmax/sound-physics-remastered#112]
**Bug description**
Game crashed when attempting to play a sound.

**Steps to reproduce the issue**
1. Unknown, happened when mobs were infighting (Vindicator + Barako)

**Expected behavior**
Game shouldn't crash when a sound gets mangled.

**Log files**
Crash:  https://pastebin.com/kh9CYXg0
Annotated Log:  https://pastebin.com/P4c7EAKR

**Versions**
- Minecraft version - 1.18.2
- Forge/Fabric version - 40.2.4
- Mod version - soundphysics-forge-1.18.2-1.0.6.jar

**Other mods**
Mowzie's Mobs, Ambient Sounds 5

**Comment by henkelmax** [comment]
Does this happen without any other mods?

**Comment by Reshy** [comment]
Yes, it occurred while testing a modpack.  I haven't been able to replicate it consistently.

**Comment by henkelmax** [comment]
Can you send me the logs of that?

**Comment by Reshy** [comment]
[2023-06-01-6.log.gz](https://github.com/henkelmax/sound-physics-remastered/files/11644729/2023-06-01-6.log.gz)
Here, it's super verbose.

**Comment by henkelmax** [comment]
There are other mods installed in these logs.

**Comment by Reshy** [comment]
Yes, it only ever happened once and it was late into my modpack's development.  I wanted to report the error even if it's be hard to precisely determine the cause.  I tried replication but had difficulties, I only know it was an interaction between Mowzie's Mobs'a Barako and a Pillager with that was made aggro to it.  Something happened when the two entities interacted that caused a crash... once.  Afterwards I tried a couple more attempts but couldn't recreate.
